kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/CreateQuizSection.vue

```diff
@@ -566,13 +573,19 @@
       handleActiveSectionAction(opt) {
         const section_id = this.activeSection.section_id;
         const editRoute = this.$router.getRoute(PageNames.QUIZ_SECTION_EDITOR, { section_id });
+        const resourcesRoute = this.$router.getRoute(PageNames.QUIZ_SELECT_RESOURCES, {
```

Comment:
@AllanOXDi, No blocker, just a clarification on whether this route would not need the optional `topic_id` param too?

Reply:
> Update this handleActiveSectionAction [method](https://github.com/learningequality/kolibri/blob/d882e51143637f2a577f20e5ba3ff5bfedbbdb56/kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/CreateQuizSection.vue#L512) so that it also checks against the string used for the label and routes the user to the select QUIZ_SELECT_RESOURCES route

No, The `topic_id `is optional.  From my understand of the above.` ResourceSelection` component should be rendered when navigating to QUIZ_SELECT_RESOURCES route which  expects an optional topic_id parameter and it can be accessed within the ResourceSelection component.